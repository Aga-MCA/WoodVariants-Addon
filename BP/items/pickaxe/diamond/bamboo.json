{"format_version":"1.21.0","minecraft:item":{"description":{"identifier":"aga_wood_variants:diamond_bamboo_pickaxe","menu_category":{"category":"equipment","group":"itemGroup.name.pickaxe"}},"components":{"minecraft:max_stack_size":1,"minecraft:hand_equipped":true,"minecraft:durability":{"max_durability":1561},"minecraft:damage":{"value":5},"minecraft:enchantable":{"value":10,"slot":"pickaxe"},"minecraft:icon":"diamond_bamboo_pickaxe","minecraft:tags":{"tags":["minecraft:digger","minecraft:is_pickaxe","minecraft:is_tool","aga:durability","minecraft:diamond_tier","minecraft:transformable_items"]},"minecraft:digger":{"use_efficiency":true,"destroy_speeds":[{"block":{"tags":"q.any_tag('stone','wood_pick_diggable','destroy:pickaxe','destroy:pickaxe:0','minecraft:is_pickaxe_item_destructible','minecraft:is_tool_item_destructible','minecraft:is_digger_item_destructible','minecraft:wood_tier_destructible')"},"speed":8},{"block":"minecraft:ice","speed":8},{"block":"minecraft:anvil","speed":8},{"block":"minecraft:bone_block","speed":8},{"block":"minecraft:mob_spawner","speed":8},{"block":"minecraft:smooth_stone","speed":8},{"block":"minecraft:enchanting_table","speed":8},{"block":"minecraft:end_bricks","speed":8},{"block":"minecraft:cracked_polished_blackstone_bricks","speed":8},{"block":"minecraft:nether_brick","speed":8},{"block":"minecraft:cracked_nether_bricks","speed":8},{"block":"minecraft:purpur_block","speed":8},{"block":"minecraft:purpur_stairs","speed":8},{"block":"minecraft:end_brick_stairs","speed":8},{"block":"minecraft:stone_slab2","speed":8},{"block":"minecraft:stone_slab3","speed":8},{"block":"minecraft:stone_brick_stairs","speed":8},{"block":"minecraft:mossy_stone_brick_stairs","speed":8},{"block":"minecraft:polished_blackstone_bricks","speed":8},{"block":"minecraft:polished_blackstone_stairs","speed":8},{"block":"minecraft:blackstone_wall","speed":8},{"block":"minecraft:blackstone_wall","speed":8},{"block":"minecraft:polished_blackstone_wall","speed":8},{"block":"minecraft:sandstone","speed":8},{"block":"minecraft:grindstone","speed":8},{"block":"minecraft:smooth_stone","speed":8},{"block":"minecraft:brewing_stand","speed":8},{"block":"minecraft:chain","speed":8},{"block":"minecraft:lantern","speed":8},{"block":"minecraft:soul_lantern","speed":8},{"block":"minecraft:quartz_ore","speed":8},{"block":"minecraft:netherrack","speed":8},{"block":"minecraft:basalt","speed":8},{"block":"minecraft:polished_basalt","speed":8},{"block":"minecraft:stonebrick","speed":8},{"block":"minecraft:warped_nylium","speed":8},{"block":"minecraft:crimson_nylium","speed":8},{"block":"minecraft:end_stone","speed":8},{"block":"minecraft:ender_chest","speed":8},{"block":"minecraft:quartz_block","speed":8},{"block":"minecraft:quartz_stairs","speed":8},{"block":"minecraft:quartz_bricks","speed":8},{"block":"minecraft:quartz_stairs","speed":8},{"block":"minecraft:nether_gold_ore","speed":8},{"block":"minecraft:furnace","speed":8},{"block":"minecraft:blast_furnace","speed":8},{"block":"minecraft:lit_furnace","speed":8},{"block":"minecraft:blast_furnace","speed":8},{"block":"minecraft:blackstone","speed":8},{"block":{"tags":"q.any_tag('stone_pick_diggable','destroy:pickaxe:1','minecraft:stone_tier_destructible')"},"speed":8},{"block":"minecraft:lapis_block","speed":8},{"block":"minecraft:copper_block","speed":8},{"block":"minecraft:iron_block","speed":8},{"block":{"tags":"q.any_tag('iron_pick_diggable','metal','destroy:pickaxe:2','minecraft:iron_tier_destructible')"},"speed":8},{"block":{"tags":"q.any_tag('diamond_pick_diggable','destroy:pickaxe:3','minecraft:diamond_tier_destructible')"},"speed":8}]}}}}