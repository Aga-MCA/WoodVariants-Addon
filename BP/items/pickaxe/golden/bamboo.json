{"format_version":"1.21.0","minecraft:item":{"description":{"identifier":"aga_wood_variants:golden_bamboo_pickaxe","menu_category":{"category":"equipment","group":"itemGroup.name.pickaxe"}},"components":{"minecraft:max_stack_size":1,"minecraft:hand_equipped":true,"minecraft:durability":{"max_durability":31},"minecraft:damage":{"value":2},"minecraft:enchantable":{"value":10,"slot":"pickaxe"},"minecraft:icon":"golden_bamboo_pickaxe","minecraft:tags":{"tags":["minecraft:digger","minecraft:is_pickaxe","minecraft:is_tool","aga:durability","minecraft:golden_tier"]},"minecraft:digger":{"use_efficiency":true,"destroy_speeds":[{"block":{"tags":"q.any_tag('stone','wood_pick_diggable','destroy:pickaxe','destroy:pickaxe:0','minecraft:is_pickaxe_item_destructible','minecraft:is_tool_item_destructible','minecraft:is_digger_item_destructible','minecraft:wood_tier_destructible')"},"speed":12},{"block":"minecraft:ice","speed":12},{"block":"minecraft:anvil","speed":12},{"block":"minecraft:bone_block","speed":12},{"block":"minecraft:mob_spawner","speed":12},{"block":"minecraft:smooth_stone","speed":12},{"block":"minecraft:enchanting_table","speed":12},{"block":"minecraft:end_bricks","speed":12},{"block":"minecraft:cracked_polished_blackstone_bricks","speed":12},{"block":"minecraft:nether_brick","speed":12},{"block":"minecraft:cracked_nether_bricks","speed":12},{"block":"minecraft:purpur_block","speed":12},{"block":"minecraft:purpur_stairs","speed":12},{"block":"minecraft:end_brick_stairs","speed":12},{"block":"minecraft:stone_slab2","speed":12},{"block":"minecraft:stone_slab3","speed":12},{"block":"minecraft:stone_brick_stairs","speed":12},{"block":"minecraft:mossy_stone_brick_stairs","speed":12},{"block":"minecraft:polished_blackstone_bricks","speed":12},{"block":"minecraft:polished_blackstone_stairs","speed":12},{"block":"minecraft:blackstone_wall","speed":12},{"block":"minecraft:blackstone_wall","speed":12},{"block":"minecraft:polished_blackstone_wall","speed":12},{"block":"minecraft:sandstone","speed":12},{"block":"minecraft:grindstone","speed":12},{"block":"minecraft:smooth_stone","speed":12},{"block":"minecraft:brewing_stand","speed":12},{"block":"minecraft:chain","speed":12},{"block":"minecraft:lantern","speed":12},{"block":"minecraft:soul_lantern","speed":12},{"block":"minecraft:quartz_ore","speed":12},{"block":"minecraft:netherrack","speed":12},{"block":"minecraft:basalt","speed":12},{"block":"minecraft:polished_basalt","speed":12},{"block":"minecraft:stonebrick","speed":12},{"block":"minecraft:warped_nylium","speed":12},{"block":"minecraft:crimson_nylium","speed":12},{"block":"minecraft:end_stone","speed":12},{"block":"minecraft:ender_chest","speed":12},{"block":"minecraft:quartz_block","speed":12},{"block":"minecraft:quartz_stairs","speed":12},{"block":"minecraft:quartz_bricks","speed":12},{"block":"minecraft:quartz_stairs","speed":12},{"block":"minecraft:nether_gold_ore","speed":12},{"block":"minecraft:furnace","speed":12},{"block":"minecraft:blast_furnace","speed":12},{"block":"minecraft:lit_furnace","speed":12},{"block":"minecraft:blast_furnace","speed":12},{"block":"minecraft:blackstone","speed":12}]}}}}